Sources/_NIOConcurrency/Helpers.swift

```diff
@@ -0,0 +1,33 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+
+// note: We have to define the variable `hasAsyncAwait` here because if we copy this code into the property below,
+// it doesn't compile on Swift 5.0.
+#if compiler(>=5.4)
+#if compiler(>=5.4) // we cannot write this on one line with `&&` because Swift 5.0 doesn't like it...
```

Comment:
I just stumbled across this...I think you forgot to add the ` && $AsyncAwait` here.

Reply:
@theMomax good spot! thank you, I'll fix this rn

Reply:
https://github.com/apple/swift-nio/pull/1791

Reply:
I'm happy to help...thanks for the quick fix!